src/main/kotlin/me/zeroeightsix/kami/feature/module/AutoGG.kt

```diff
@@ -0,0 +1,58 @@
+package me.zeroeightsix.kami.feature.module
+
+import io.github.fablabsmc.fablabs.api.fiber.v1.annotation.Setting
+import me.zero.alpine.listener.EventHandler
+import me.zero.alpine.listener.Listener
+import me.zeroeightsix.kami.event.PacketEvent
+import me.zeroeightsix.kami.gui.text.CompiledText
+import net.minecraft.network.packet.s2c.play.GameMessageS2CPacket
+import java.util.UUID
+
+@Module.Info(
+    name = "AutoGG",
+    category = Module.Category.MISC,
+    description = "Says GG or a custom message at the end of a match"
+)
+object AutoGG : Module() {
+    @Setting
+    var ggMessage = CompiledText(mutableListOf())
+
+    private const val updateLimit: Long = 2000L // 2 Seconds
```

Comment:
I doubt people will be winning games within 2 seconds of each other. You can make this significantly bigger.

Reply:
The purpose of the cooldown is to not have it re-trigger in the time it takes the server to send all the messages.

Reply:
Yes, but you can avoid checking other messages for free by making this bigger anyways.